docs/docs/manual/exploring.md

```diff
@@ -1,102 +1,60 @@
 
- (only current view)
+An “error” data type is created when the cell is storing an error generated during a transformation in OpenRefine.
 
+A “null” data type is a special value which basically means “this cell has no value.” It’s used to differentiate between cells that have values such as “0” or “false” - or a cell that looks empty but has, for example, spaces in it. When you use `type(value)`, it will show you that the cell’s value is “null” and its type is “undefined.” You can opt to [show “null” values](#view) to differentiate them from empty strings, by going to “All” > “View” > “Show/Hide ‘null’ values in cells.”
 
-### Regular expressions
+Converting a cell's data type is not the same operation as transforming its contents. For example, using a column-wide transform such as “Transform” > “Common transforms …” > “to date” may not convert all values successfully, but going to an individual cell, clicking “edit” and changing the data type can successfully convert text to a date. These operations use different underlying code. 
 
+To transform data from one type to another, see [Transforming data](transforming#transform) for information on using common tranforms, and see [Expressions](expressions) for information on using `toString()`, `toDate()`, and other functions. 
 
-## Sort and View
+## Row types: rows vs. records
 
+A row is a simple way to organize data: a series of cells, one cell per column. Sometimes there are multiple pieces of information in one cell, such as when a survey respondent can select more than one response. In cases where there is more than one value for a single column in one or more rows, you may wish to use OpenRefine’s records mode: this defines a single record (a survey response, for example) as potentially containing more than one row. From there you can transform cells into multiple rows, each cell containing one value you’d like to work with. 
 
-### Sort
+Generally, when you import some data, OpenRefine reads that data in row mode. From there you can convert the project into records mode. OpenRefine remembers this action and will present you with records mode each time you open the project from then on. 
 
+OpenRefine understands records based on the content of the first column, what we call the "key column." Splitting a row into a multi-row record will base all association on the first column in your dataset. If you have more than one column to split out into multiple rows, OpenRefine will keep your data associated with its original record: you can imagine this structure as a tree with many branches, all leading back to the same trunk. 
 
-### View: Collapsing (and expanding)
+For example, your key column may be a family name, with multiple family members identified by their first names, associated to that family name. You may have one or more phone numbers listed for each person. The phone numbers are linked to the first names, which are linked to the last name. 
```

Comment:
I'd suggest that an ID of some kind would be a better example of a key column - the scenario you describe would work in a sense I think a unique ID that links the rows together is a better example of how records work

Reply:
I'm not sure why you want a unique ID for this. I think this example works well with just the film title, not the ID?

|IMDB ID|Work|Actor|Role|
|---|---|---|---|
|tt0032138|The Wizard of Oz|Judy Garland|Dorothy Gale|
|||Ray Bolger|"Hunk"|
||||The Scarecrow|
|||Jack Haley|"Hickory"|
||||The Tin Man|
|||Bert Lahr|"Zeke"|
||||The Cowardly Lion|
|||Frank Morgan|Professor Marvel|
||||The Gatekeeper|
||||The Carriage Driver|
||||The Guard|
||||The Wizard of Oz|
|||Margaret Hamilton|Miss Almira Gulch|
||||The Wicked Witch of the West|

Reply:
I've removed the unique ID for now - maybe the section needs more explication about how you don't actually need a key column of unique IDs (because OR does that work behind the scenes), but of course your data may need to be sorted (or un-sorted) not to confuse the issue. But I think that can wait for a second pass.